pkg/operator/hive/hive_controller.go

```diff
@@ -525,6 +533,9 @@ func (r *ReconcileHiveConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 	// Incorporate the AWSPrivateLink configmap hash
+	confighash = computeHash("", confighash, awsPlConfigHash)
+
+	// Incorporate the PrivateLink configmap hash
 	confighash = computeHash("", confighash, plConfigHash)
```

Comment:
the hashes arg is variadic, so we can combine these:
```suggestion
	// Incorporate the [AWS]PrivateLink configmap hashes
	confighash = computeHash("", confighash, awsPlConfigHash, plConfigHash)
```
That said, we _could_ check which controller is active by inspecting the controllersConfig and only process (above) and incorporate the hash (here) for the active one. The benefit being that we don't reroll all the hive pods if the inactive one changes. I don't know if we want to try to be that clever though :)